Meetings/Meeting.ics

```diff
@@ -1,67 +1,150 @@
 BEGIN:VCALENDAR
 PRODID:-//Microsoft Corporation//Outlook 16.0 MIMEDIR//EN
 VERSION:2.0
-METHOD:PUBLISH
+METHOD:REQUEST
```

Comment:
I seem to be unable to "import" this meeting. It seems that my outlook is under the impression that this is an invitation, and I am not invited.
(Maybe the problem is "me", so probably someone else should re-check this.)

Reply:
I see the same. Outlook completely blocks the accept/decline buttons.

Reply:
Notably, it looks like you can click "No Response Required" and it will add it to your calendar anyways

Reply:
Sorry, we'll fix it later.